tests/PHPStan/Command/ErrorFormatter/TableErrorFormatterTest.php

```diff
@@ -217,6 +217,10 @@ public function testEditorUrlWithTrait(): void
 
 	public function testEditorUrlWithRelativePath(): void
 	{
+		if ('JetBrains-JediTerm' === getenv('TERMINAL_EMULATOR')) {
+			$this->markTestSkipped('PHPStorm console does not support links in console.');
```

Comment:
1) I don't like yoda conditions
2) It's `PhpStorm`.

Reply:
thanks, fixed